WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -376,117 +409,167 @@ - (void)configureNoResultsView
     // Refresh the NoResultsView Properties
     self.noResultsView.titleText = self.noResultsTitleText;
 
-    // Only add and animate no results view if it isn't already
-    // in the table view
-    if (![self.noResultsView isDescendantOfView:self.tableView]) {
-        [self.tableView addSubviewWithFadeAnimation:self.noResultsView];
+    // Only add and animate no results view if it isn't already in the table view
+    if (![self.noResultsView isDescendantOfView:self.view]) {
+        [self.view addSubviewWithFadeAnimation:self.noResultsView];
     } else {
         [self.noResultsView centerInSuperview];
     }
 
-    [self.tableView sendSubviewToBack:self.noResultsView];
+    [self.view bringSubviewToFront:self.noResultsView];
 }
 
-- (NSString *)noResultsTitleText
-{
-    if (self.syncHelper.isSyncing) {
-        return NSLocalizedString(@"Fetching comments...", @"A brief prompt shown when the comment list is empty, letting the user know the app is currently fetching new comments.");
-    }
 
-    return NSLocalizedString(@"Be the first to leave a commment.", @"Message shown encouraging the user to leave a comment on a post in the reader.");
+#pragma mark - Autolayout Helpers
+
+- (void)updateViewConstraints
+{
```

Comment:
Performance-wise, it takes roughly 1 millisecond on the iPhone 6 to execute this method, and around 5ms in an iPad Mini (1st gen).

However, i'm not entirely sure if the autolayout machinery actually evaluates constraints asynchronously, or synchronously. I'll be updating this code, i've come up with another idea, perhaps way cleaner: simply (alway) attach Reply + Suggestions, and hide / enable, as needed, in the refresh call, with a little hacky trick.

Thanks Eric!